tests/unit/splinter_test.c

```diff
@@ -667,6 +667,48 @@ CTEST2(splinter, test_splinter_print_diags)
    allocator_print_stats(alp);
    allocator_print_allocated(alp);
 
+   CTEST_LOG_INFO("\n** Trunk nodes filter metapages list: "
+                  "trunk_print_root_nodes_filter_metapages() **\n");
+
+   // Exercise print method of mini-allocator's unkeyed meta-page
+   trunk_print_root_nodes_filter_metapages(Platform_default_log_handle, spl);
+
+   trunk_destroy(spl);
+}
+
+/*
+ * -----------------------------------------------------------------------------
+ * Test case to exercise verbose print logging instrumentation in trunk.c
+ * generated by trunk_default_log_if_enabled().
+ * -----------------------------------------------------------------------------
+ */
+CTEST2(splinter, test_splinter_verbose_print_diags)
```

Comment:
Added new test case to exercise the calls to `trunk_default_log_if_enabled()` at different code points, which prints intermediate state of trunk trees while large insertions, and other activities (like compaction etc.) are going on.